contrib/sssd.spec.in

```diff
@@ -1097,11 +1097,12 @@ install -D -p -m 0644 %{SOURCE1} %{buildroot}%{_sysusersdir}/sssd.conf
 
 %if %{use_sssd_user}
 %pre common
+! getent passwd sssd >/dev/null || usermod sssd -d /run/sssd >/dev/null || true
```

Comment:
Hi,

I wonder if something like
```
! getent passwd sssd | grep '/:/' >/dev/null || usermod sssd -d /run/sssd >/dev/null || true
```
(not tested) would help to call `usermod` not during all updates but only once?

bye,
Sumit

Reply:
JFTR: `usermod` works fine if home dir isn't changed, but ok, I'll update this.

Reply:
keep in mind that you need to add grep to required packages if you use it

Reply:
Ah, I always forget that `grep` is not part of `coreutils`.  So I'm inclined to withdraw the comment. Running usermod on all updates might be the lesser pain compared to having another dependency. So feel free to resolve this conversation, if you agree.

Reply:
Then let's keep it "running on all updates".